sdk/storage/storage-blob/README.md

```diff
@@ -45,9 +30,16 @@ The preferred way to install the Azure Storage Blob client library for JavaScrip
 npm install @azure/storage-blob
 ```
 
-### Authenticating with Azure Active Directory
+### Authenticate the client
```

Comment:
`the client` and `In order to interact with the Storage Service you'll need to create an instance of the Service Client class.` sound like we only support authenticating the service client but actually service/container/blob clients are all supported. Please consider some rewording.